source/extensions/tracers/common/ot/opentracing_driver_impl.h

```diff
@@ -43,6 +43,9 @@ class OpenTracingSpan : public Tracing::Span, Logger::Loggable<Logger::Id::traci
   std::string getBaggage(absl::string_view key) override;
   void setBaggage(absl::string_view key, absl::string_view value) override;
 
+  // TODO: This method is unimplemented for OpenTracing.
+  std::string getTraceId() const override { return std::string(); };
```

Comment:
nit: use `EMPTY_STRING` here and in other non-test places

Reply:
Done